spinalcordtoolbox/utils/shell.py

```diff
@@ -71,6 +72,11 @@ def display_viewer_syntax(files, verbose, im_types=[], minmax=[], opacities=[],
     display_viewer_syntax([file1, file2, file3])
     display_viewer_syntax([file1, file2], im_types=['anat', 'softseg'], minmax=['', '0,1'], opacities=['', '0.7'])
     """
+    # If the file's absolute path is relative to the CWD, then use just the relative path to that file.
+    files = [str(abspath.relative_to(Path.cwd())) if abspath.is_relative_to(Path.cwd())
+             # Otherwise, the file must be outside the working directory, so use the absolute path.
+             else str(abspath) for abspath in [Path(f).absolute() for f in files]]
```

Comment:
I think a nested list comprehension (especially involving an `if`-expression) is a sign that this particular operation should be turned into a small utility function.